projects/core/src/cart/occ/occ-cart.adapter.ts

```diff
@@ -0,0 +1,111 @@
+import { Injectable } from '@angular/core';
+import { CartAdapter } from '../connectors/cart/cart.adapter';
+import { Observable, throwError } from 'rxjs';
+import { Cart, CartList } from '../../occ/occ-models/occ.models';
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { catchError, map, pluck } from 'rxjs/operators';
+import { OccEndpointsService } from '../../occ/services/occ-endpoints.service';
+import { ConverterService } from '../../util/converter.service';
+import { CART_NORMALIZER } from '../connectors/cart/converters';
+
+// for mini cart
+const BASIC_PARAMS =
+  'DEFAULT,deliveryItemsQuantity,totalPrice(formattedValue),' +
+  'entries(totalPrice(formattedValue),product(images(FULL)))';
+
+// for cart details page
+const DETAILS_PARAMS =
```

Comment:
while we're here, can we move the endpoint configuration to the new config?

Reply:
There is a separate ticket waiting for doing this part AFAIK